scripts/test/lib/cluster.sh

```diff
@@ -18,6 +19,19 @@ function load_cluster_details() {
   echo "VPC ID: $VPC_ID, Service Role ARN: $SERVICE_ROLE_ARN, Role Name: $ROLE_NAME"
 }
 
+function load_deveks_cluster_details() {
+
+  PROVIDER_ID=$(kubectl get nodes -ojson | jq -r '.items[0].spec.providerID')
+  INSTANCE_ID=${PROVIDER_ID##*/}
+  VPC_ID=$(aws ec2 describe-instances --instance-ids ${INSTANCE_ID} --no-cli-pager | jq -r '.Reservations[].Instances[].VpcId')
```

Comment:
You can get VPC ID directly from `aws eks describe-cluster --name $CLUSTER_NAME --endpoint $ENDPOINT --query 'cluster.resourcesVpcConfig.vpcId'`

Reply:
This is cluster created with deveks, which doesn't get registered on this api. That's why the new function for deveks tests

Reply:
Makes sense. Thanks.